laos_climate_change/data_loaders.py

```diff
@@ -0,0 +1,27 @@
+# Verify world shapefile
+from os.path import exists
+from urllib.request import urlretrieve
+
+path_to_file = "C:/Users/camil/Dropbox/Servicios/OECD-GGGI/laos-climate-change/laos_climate_change/wb_countries_admin0_10m.zip"
+url = "https://datacatalogfiles.worldbank.org/ddh-published/0038272/DR0046659/wb_countries_admin0_10m.zip?versionId=2024-05-14T14:58:01.5696428Z"
+filename = "wb_countries_admin0_10m.zip"
+
+
+def download_shape_files():
+    if not exists(path_to_file):
```

Comment:
You also need to create the directory if it doesn't exist. You can use `os.makedirs`

Reply:
Addressed in the latest commit!